### PR TITLE
ICP-6497 Neo Coolcam Dual Switch: added refresh support to second end…

### PR DESCRIPTION
**Pull request by greens**
…point (#3582)

* ICP-6497 added refresh support to second endpoint

### CODE REVIEW

**Comment by greens** [comment]
Requested to get this directly to staging. Has been tested in ST-Pol.

File: devicetypes/smartthings/zwave-dual-switch.src/zwave-dual-switch.groovy [hunk 1]
```diff
@@ -65,7 +65,7 @@ def installed() {
 	} catch (e) {
 		log.warn "Failed to add endpoint 2 ($desc) as Z-Wave Binary Switch Endpoint - $e"
 	}
-	configure()
+	response(configure())
```
Comment (tpmanley) [inline devicetypes/smartthings/zwave-dual-switch.src/zwave-dual-switch.groovy:68]: Configure already returns a response

File: devicetypes/smartthings/zwave-dual-switch.src/zwave-dual-switch.groovy [hunk 2]
```diff
@@ -173,7 +173,10 @@ def ping() {
 def refresh() {
 	// parent DTH controls endpoint 1
 	def endpointNumber = 1
```
Comment (tpmanley) [inline devicetypes/smartthings/zwave-dual-switch.src/zwave-dual-switch.groovy:175]: Can remove this now

File: devicetypes/smartthings/zwave-dual-switch.src/zwave-dual-switch.groovy [hunk 3]
```diff
@@ -173,7 +173,10 @@ def ping() {
...
-	encap(endpointNumber, zwave.switchBinaryV1.switchBinaryGet())
+	def cmdmap = [] 
+	cmdmap << encap(1, zwave.switchBinaryV1.switchBinaryGet())
```
Comment (tpmanley) [inline devicetypes/smartthings/zwave-dual-switch.src/zwave-dual-switch.groovy:177]: Could simply this to just `[encap..., encap...]`

**Comment by greens** [comment]
it's a cherry-pick that's already been QA'd and merged to master, but I can change it if you want.

**Comment by tpmanley** [comment]
Didn't realize that. It's fine to leave it to avoid conflicts later on the merge.

**Review by tpmanley — APPROVED** [review]

**Pull request merged**